core-common/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -146,4 +146,17 @@ MappingInheritanceStrategy mappingInheritanceStrategy()
      * @return strategy how to do null checking
      */
     NullValueCheckStrategy nullValueCheckStrategy() default ON_IMPLICIT_CONVERSION;
+
+    /**
+     * If MapStruct could not find another mapping method or apply an automatic conversion it will try to generate a
+     * forged mapping method for the properties. If this property is set to {@code true} MapStruct will not try to
+     * automatically generate forged mapping methods.
+     *
+     * Can be overridden by {@link Mapper#disableForgedMappingMethodsGeneration()}
+     *
+     * @return whether the automatic generation of forged mapping methods is disabled
+     *
+     * @since 1.2.0
+     */
+    boolean disableForgedMappingMethodsGeneration() default false;
```

Comment:
has the term "forged" leaked into the documentation? I thought it was for internal use only...